build.sbt

```diff
@@ -2,13 +2,21 @@ lazy val commonSettings = Seq(
   organization       := "ch.megard",
   version            := "0.0.0-SNAPSHOT",
   scalaVersion       := "2.13.10",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.17", "3.2.2"),
+  crossScalaVersions := Seq(scalaVersion.value, "2.12.17", "3.3.0"),
   scalacOptions ++= Seq(
     "-encoding",
     "UTF-8",
     "-unchecked",
     "-deprecation"
-  ),
+  ) ++ (CrossVersion.partialVersion(scalaVersion.value) match {
+    case Some((2, n)) if n == 12 || n == 13 =>
+      Seq(
+        "-opt:l:inline",
```

Comment:
For Scala 2/Scala 3 the `@inline` annotation doesn't actually do anything unless you enable it. `-opt-inline-from:<sources>` was also added since this is safe to do in libraries.